tsdb/index/index.go

```diff
@@ -523,6 +523,11 @@ func (w *Writer) AddSymbol(sym string) error {
 }
 
 func (w *Writer) finishSymbols() error {
+	// Right now, the index symbol_table's <len> part is 4byte length. So the total symbol string length must less than 4byte
+ 	if w.f.pos - w.toc.Symbols - 4 > 4294967295 {
```

Comment:
Since we are using `w.f.pos - w.toc.Symbols - 4` in multiple places, how about we have a `symbolTableSize := w.f.pos - w.toc.Symbols - 4` above and re-use it in all 3 places here?